server/controllers/content.js

```diff
@@ -35,7 +35,8 @@ export const renderArticle = async(ctx, next) => {
           category: 'editorial',
           canonicalUri: `${ctx.globals.rootDomain}/articles/${id}`
         }), getEditorialAnalyticsInfo(article)),
-        article: article
+        article: article,
+        isPreview: preview
```

Comment:
Could you name the `preview` variable to `isPreview` at instantiation, it's a nicer name?

Reply:
Sure thing